web_widget_many2many_tags_multi_selection/README.rst

```diff
@@ -0,0 +1,46 @@
+Allows multiple selection on many2many_tags widget
```

Comment:
Updated to latest README template: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Reply:
Done ! Hope I haven't forgotten anything.
I amended the original commit & force pushed to avoid adding new commits to this PR.

Reply:
@MaxyMoos Could you put the title in capital letters?  😉